static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -393,20 +395,23 @@ class GroupDetails extends Component<Props, State> {
     GroupStore.onPopulateReleases(this.props.params.groupId, releases);
   }
 
-  async fetchReplaysCount() {
+  async fetchReplayIds() {
     const {api, location, organization, params} = this.props;
     const {groupId} = params;
 
-    this.setState({loadingReplaysCount: true});
+    this.setState({loadingReplayIds: true});
 
-    const eventView = EventView.fromSavedQuery({
-      id: '',
-      name: `Replays in issue ${groupId}`,
-      version: 2,
-      fields: ['count()'],
-      query: `issue.id:${groupId}`,
-      projects: [],
-    });
+    const eventView = EventView.fromNewQueryWithLocation(
+      {
+        id: '',
+        name: `Errors within replay`,
+        version: 2,
+        fields: ['id', 'replayId'],
+        query: `issue.id:${groupId} !replayId:""`,
+        projects: [],
```

Comment:
Does EventView add the current project when querying?

If not: TODO: add current issue detail's project here

Reply:
it gets added from the `location`. I can check to see if this field needs to be typed out here and put a shot of the final url that it builds

Reply:
Here's the query: `projects` and `statsPeriod` are coming from `location`:
<img width="297" alt="Screen Shot 2022-09-06 at 3 37 29 PM" src="https://user-images.githubusercontent.com/187460/188752556-ad79f0df-b07d-4143-aa5a-22e29f2a2d7d.png">

I need to type `projects: []` to satisfy TS